axes/middleware.py

```diff
@@ -0,0 +1,30 @@
+from axes.attempts import is_already_locked
+from axes.utils import get_lockout_message
+from django.contrib.auth.backends import ModelBackend
+from django.core.exceptions import PermissionDenied
```

Comment:
A more traditional import order would be to import Python core packages first, then Django packages and then custom packages. Something like

    from django import ...

    from axes import ...